scripts/prometheus-node-exporter/nodestats-tomesh.py

```diff
@@ -0,0 +1,77 @@
+import os 
```

Comment:
Tabs in this file should be spaces.

Reply:
```cat scripts/prometheus-node-exporter/nodestats-tomesh.py  | grep -P '\t' ```

Returns nothing. What tabs am I missing?